### ISSUE TITLE
Inconsistent casing

### ISSUE THREAD

**Issue by Delagen** [ReactiveX/rxjs#2876]
**RxJS version:** 5.5.0-beta.2

**Code to reproduce:**

**Expected behavior:**
No errors

**Actual behavior:**
TS1149: File name '.../rxjs/observable/of.d.ts' differs from already included file name '.../rxjs/Observable/of.d.ts' only in casing.

**Additional information:**
I think it caused by compilation store files in lowercase folder, but typings compiled in uppercase folder

**Comment by Delagen** [comment]
I think it windows specific.
But now error are gone. Strange.

**Comment by Delagen** [comment]
It occurs when compiler use it default settings.

**Comment by Delagen** [comment]
I use the following in build task, and have no error, but IDE tools reporting it
```
var ts = require("typescript");
ts.sys.useCaseSensitiveFileNames = true;
```

**Comment by benlesh** [comment]
closed by #2878
